### DISCUSSION TITLE
Is this course a good start in ML?

### DISCUSSION THREAD

**Discussion by alexcaldarone** [OpenMined/courses#399] [Course: Foundations of Private Computation] [answered]
Hi,
I am currently studying statistics at university and have some programming experience in Python. I am looking to get into ML and eventually privacy-preserving ML.
The question is: Is this course good as a first course in ML, or do you advise to first take a more general course (such as fast.ai or similar) and then come back to take this one?

**Comment by adilsheraz** [comment, marked as answer]
No I don't think so to start with ML from here. I suggest do start with traditional ML after getting some expertise you can jump over here.
